### PR TITLE
Fix issue 1905.

### LINKED ISSUE #1905: Bad naming style
**What would you like to be added/modified**:
The name of functions, variables is not appropriate.
file: kubeedge/edge/pkg/devicetwin/dtmanager/membership.go
1. Function name needn't use past tense. Not only this function, but also many other function names should be refined.
Eg:
line 164
// Added add device to the edge group
func Added

2. Bad variable name. Here deviceModel -> deviceInstance. For deviceModel and deviceinstance are two definitions in device manager.
deviceExist  -> isDeviceExist.
line 	178:	deviceModel, deviceExist := context.GetDevice(device.ID)

**Why is this needed**:
Good coding style is the base.

### PR DESCRIPTION
**Pull request by sailorvii**
Changes:
1. Refine some function names. Those function names use past tense which are unreasonable.
   Refine them to present tense.
2. Refine some variable names. Device and device model are two different definitions. In
   this case, it's device.

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #1905

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
NONE
```release-note

```

### CODE REVIEW

File: edge/pkg/devicetwin/dtmanager/device_test.go [hunk 1]
```diff
@@ -263,8 +263,8 @@ func TestDealDeviceStateUpdate(t *testing.T) {
 	}
 }
 
-//TestDealDeviceUpdated is function to test dealDeviceUpdated().
-func TestDealDeviceUpdated(t *testing.T) {
+//TestDealUpdateDeviceAttr is function to test dealUpdateDeviceAttr().
```
Comment (daixiang0) [inline edge/pkg/devicetwin/dtmanager/device_test.go:266]: Useless comment here.

Comment (sailorvii) [reply edge/pkg/devicetwin/dtmanager/device_test.go]: Done. Remove all useless comments.

File: edge/pkg/devicetwin/dtmanager/device_test.go [hunk 2]
```diff
@@ -102,7 +102,7 @@ func TestDeviceStartAction(t *testing.T) {
 	}
 }
 
-// TestDevicetHeartBeat is function to test Start() when value is passed in HeartBeatChan.
+// TestDeviceHeartBeat is function to test Start() when value is passed in HeartBeatChan.
```
Comment (fisherxu) [inline edge/pkg/devicetwin/dtmanager/device_test.go:105]: TestDeviceStartHeartBeat?

Comment (sailorvii) [reply edge/pkg/devicetwin/dtmanager/device_test.go:105]: Thanks for your review. 

I thought the original name is better. The function Start/Run is the entry for almost all processes. It's a little weird to add this word.

File: edge/pkg/devicetwin/dtmanager/device_test.go (hunk 1 above)
Comment (sailorvii) [reply edge/pkg/devicetwin/dtmanager/device_test.go]: PTAL @daixiang0

File: edge/pkg/devicetwin/dtmanager/device_test.go (hunk 2 above)
Comment (fisherxu) [reply edge/pkg/devicetwin/dtmanager/device_test.go:105]: Sounds good, so the func name in L106 need to be fixed?

File: Makefile [hunk 3]
```diff
@@ -288,3 +288,10 @@ bluetoothdevice: clean
 .PHONY: bluetoothdevice_image
 bluetoothdevice_image:bluetoothdevice
 	docker build -t bluetooth_mapper:v1.0 ./mappers/bluetooth_mapper/
+
+.PHONY: modbusmapper
+modbusmapper: clean
+	hack/make-rules/modbusmapper.sh
+.PHONY: modbusmapper_image
+modbusmapper_image:modbusmapper
+	sudo docker build -t modbusmapper:v1.0 ./mappers/modbustcp_mapper
```
Comment (fisherxu) [inline Makefile:297]: These changes should be in another PR, thanks

File: hack/lib/mapper.sh [hunk 4]
```diff
@@ -28,3 +28,13 @@ kubeedge::mapper::build_bluetoothdevice() {
...
+
+kubeedge::mapper::build_modbusmapper() {
+  modbus_binary=${KUBEEDGE_GO_PACKAGE}/mappers/modbustcp_mapper
+  name=modbustcp_mapper
+
+  mkdir -p ${KUBEEDGE_OUTPUT_BINPATH}
+  set -x
+  go build -o ${KUBEEDGE_OUTPUT_BINPATH}/${name} $modbus_binary
+  set +x
+}
```
Comment (fisherxu) [inline hack/lib/mapper.sh:40]: ditto

File: edge/pkg/devicetwin/dtmanager/device_test.go (hunk 2 above)
Comment (sailorvii) [reply edge/pkg/devicetwin/dtmanager/device_test.go:105]: Yes, you're right. Refined it.

Thanks.

**Comment by fisherxu** [comment]
/lgtm
Thanks.

**Pull request merged**